src/styles/active-state.scss

```diff
@@ -0,0 +1,14 @@
+.is-active {
```

Comment:
I think this classes should stay with "nav" block. We had to name this class "is-active" only in this task while according to BEM it should be named (nav__link--is-active). So technically this refers to "nav.scss" file.